### PR TITLE
Create an easy log formatter that can be used on demand to debug things

### PR DESCRIPTION
**Pull request by Deorigosa**
...by using
```ruby
config.logger = WhateverLogger.new(STDWHEREVER)
config.logger.formatter = Technologic::SimpleLogFormatter.new
```

### CODE REVIEW

File: technologic/lib/technologic/version.rb [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 module Technologic
   # This constant is managed by spicerack
-  VERSION = "0.15.1"
+  VERSION = "0.15.2pre"
```
Comment (garside) [inline technologic/lib/technologic/version.rb]: Let's keep the release version stuff out of the PR. makes it easier to do once all is merged down for spice rack!

File: technologic/lib/technologic/simple_log_formatter.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      @datetime_format = "%H:%M:%S:%3N"
+    end
+
+    def call(severity, time, _, msg)
+      FORMAT % [ format_datetime(time), severity, msg2str(msg) ]
+    end
+
+    private
+
+    def msg2str(msg)
```
Comment (garside) [inline technologic/lib/technologic/simple_log_formatter.rb]: let's shy away from the terse vars and names? `format_message(message)` feels more idiomatic IMO

File: technologic/lib/technologic/simple_log_formatter.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Technologic
+  class SimpleLogFormatter < ::Logger::Formatter
```
Comment (garside) [inline technologic/lib/technologic/simple_log_formatter.rb]: Maybe give this a more "branded" name to distinguish it?

File: technologic/lib/technologic/simple_log_formatter.rb (hunk 2 above)
Comment (Deorigosa) [reply technologic/lib/technologic/simple_log_formatter.rb]: It's an override of a `::Logger#msg2str`

File: technologic/lib/technologic/simple_log_formatter.rb (hunk 3 above)
Comment (Deorigosa) [reply technologic/lib/technologic/simple_log_formatter.rb]: But why? `Technologic::SimpleLogFormatter` fully represents what it does

**Pull request closed without merging**